files/build_templates/per_namespace/lldp.service.j2

```diff
@@ -1,16 +1,29 @@
 [Unit]
 Description=LLDP container
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
+Requires=updategraph.service
+After=database@%i.service swss@%i.service syncd@%i.service
+After=updategraph.service
+{% else %}
 Requires=updategraph.service
 After=updategraph.service swss.service syncd.service
+{% endif %}
```

Comment:
See suggestion above.

Reply:
Modified as per suggestion

Reply:
I believe this can be simplified further:

```suggestion
Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service
After=database{% if multi_instance == 'true' %}@%i{% endif %}.service
After=swss{% if multi_instance == 'true' %}@%i{% endif %}.service
After=syncd{% if multi_instance == 'true' %}@%i{% endif %}.service
```

Also, why does lldp.service no longer rely on updategraph.service?

Reply:
Looks like this comment has not yet been addressed.